config/webpack/webpack-base.js

```diff
@@ -22,23 +22,38 @@ module.exports = {
     path: outputPath,
     umdNamedDefine: true
   },
-  devtool: 'source-map',
+  resolve: {
+    // Add '.ts' as resolvable extensions.
+    extensions: ['.ts', '.js'],
+    modules: ['./src', 'node_modules']
+  },
+  // A SourceMap is added as a DataUrl to the bundle.
+  devtool: '#inline-source-map',
+  // devtool: 'source-map',
   module: {
-    rules: [{
-      enforce: 'pre',
-      test: /\.js$/,
-      exclude: /(node_modules)/,
-      loader: 'eslint-loader',
-      options: {
-        failOnError: false
+    rules: [
+      // All files with a '.ts' extension will be handled by 'ts-loader'.
+      { 
+        test: /\.ts$/,
+        loader: 'ts-loader',
+        exclude: /node_modules(?!\/webpack-dev-server)/
       }
-    }, {
-      test: /\.js$/,
-      exclude: /(node_modules)/,
-      use: [{
-        loader: 'babel-loader'
-      }]
-    }]
+    ]
+    // rules: [{
+    //   enforce: 'pre',
+    //   test: /\.js$/,
+    //   exclude: /(node_modules)/,
+    //   loader: 'eslint-loader',
+    //   options: {
+    //     failOnError: false
+    //   }
+    // }, {
+    //   test: /\.js$/,
+    //   exclude: /(node_modules)/,
```

Comment:
Does ts-loader also check our eslint rules and transpile based on our babelrc/config? Is there a different config we now need to be aware of?

Reply:
I removed ts-loader and re-applied eslint-loader and babel-loader with some tweaks. So babelrc/config and etc are currently working again.